styles.css

```diff
@@ -0,0 +1,80 @@
+* {
+  margin: 0;
+  padding: 0;
+  background-color: #191765;
+  font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;
+}
+
+.toolbar {
+  min-height: 4rem;
+  display: flex;
+  align-items: center;
+  padding: 15px;
+  color: white;
+  opacity: 1;
+}
+
+.toolbar-items {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+}
+
+.bar-icon {
+  zoom: 1.5;
+}
+
+.bar-icon:hover {
+  zoom: 1.8;
+  transition: ease-out;
+  box-shadow: 1px 1px 5px gray;
+}
+
+#logo-text {
+  line-height: 2px;
+  font-weight: 500;
+}
+
+.headline-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  background: url("/headline_bg.svg");
+  background-repeat: no-repeat;
+  min-height: 40vh;
+}
+
+.headline-section > h1 {
+  color: #7d73fc;
+  font-weight: 700;
+  line-height: 50px;
+  font-size: 2.3rem;
+  padding: 1rem;
+  text-align: center;
+  background-color: transparent;
+}
+
+.headline-section > h2 {
+  color: #edecff;
+  font-weight: 500;
+  padding: 1rem;
+  text-align: center;
+  background-color: transparent;
+}
+
+.headline-section > p {
```

Comment:
- [ ] As you could see in the previous comment, the `font-weight` in this paragraph is not matching the original design. You should consider double-checking your font import and adding the `400` font weight!

Reply:
```
.headline-section > p {
  color: #c8c8d4;
  padding: 2rem;
  text-align: center;
  line-height: 20px;
  background-color: transparent;
  font-weight: 400;
}
```

`font-weight: 400;` added to paragraph styling